### DISCUSSION TITLE
Баг с функцией «Поиск дубликатов»

### DISCUSSION THREAD

**Discussion by LyokhaOhm** [Chimildic/YaMuTools#71] [Помощь с ошибкой]
**Функция  «Поиск дубликатов» удаляет все файлы — и дубликат и «оригинал»!!**
**Должно быть так, что удаляются только дубли!** 

Поясню на всякий случай:
Если файл добавленный в плейлист «первым» из «комплекта дублей» считается условно «оригиналом», 
то файлы добавленные «позднее первого» считаются условно «копиями».

Поэтому функция «Поиск дубликатов» должна удалять ТОЛЬКО КОПИИ, оставляя при этом один файл «оригинал» из «комплекта дублей». 

Но сейчас удаляется вообще всё!
**Сейчас если в тестовый плейлист добавить один и тот же трек дважды, то функция «Поиск дубликатов» полностью очищает плейлист!!**

https://user-images.githubusercontent.com/64336600/200613663-8047a5e5-d4fa-46b2-9713-9603b1407b39.mov

Файл с видео демонстраций прикреплён. 
Смотрите.

**Comment by Chimildic** [comment]
У вас получиться повторить подобное когда в плейлисте будет хотя бы один уникальный трек при остальных дубликатах? 
Просто похоже на упущение пограничного состояние конкретного случая, когда все треки одинаковые.

**Comment by sergvpr** [comment]
Еще баг с поиском дублей - не удаляет в недоступных.
Возможно, это из-за отсутствия ссылок на отсутствующие треки?
Тогда, может добавить опцию удалять по "артист + трек", также, тогда это будет полезно в плейлистах, если треки с разных альбомов/сборников.

![image](https://user-images.githubusercontent.com/74408860/215283204-e3c59040-db25-4ddb-a6fd-2cd519930bd2.png)
